.github/workflows/chromatic-push.yml

```diff
@@ -1,29 +0,0 @@
-name: Chromatic Push
```

Comment:
We need to keep this file to update the baselines after a PR is merged.
Please remove the `trigger-integration` branch we'll handle that in `cypress-push`